src/app/visit/visit.component.html

```diff
@@ -10,61 +10,64 @@
-        <th mat-header-cell *matHeaderCellDef>
-            {{ 'VISIT_TABLE.NAME' | translate }}
-        </th>
-        <td mat-cell *matCellDef="let element">
-            {{ element.name }}
-        </td>
-    </ng-container>
-    <ng-container matColumnDef="type">
-        <th mat-header-cell *matHeaderCellDef>
-            {{ 'VISIT_TABLE.TYPE' | translate }}
-        </th>
-        <td mat-cell *matCellDef="let element">
-            {{ element.type }}
-        </td>
-    </ng-container>
+<div class="mat-elevation-z8" [hidden]="!patient.name">
+    <table mat-table [dataSource]="dataSource" class="mat-elevation-z8">
+        <ng-container matColumnDef="id">
+            <th mat-header-cell *matHeaderCellDef>
+                {{ 'VISIT_TABLE.ID' | translate }}
+            </th>
+            <td mat-cell *matCellDef="let element">
+                {{ element.id }}
+            </td>
+        </ng-container>
+        <ng-container matColumnDef="negativePoint">
+            <th mat-header-cell *matHeaderCellDef>
+                {{ 'VISIT_TABLE.NEGATIVE_POINT' | translate }}
+            </th>
+            <td mat-cell *matCellDef="let element">
+                {{ element.negativePoint }}
+            </td>
+        </ng-container>
+        <ng-container matColumnDef="positivePoint">
+            <th mat-header-cell *matHeaderCellDef>
+                {{ 'VISIT_TABLE.POSITIVE_POINT' | translate }}
+            </th>
+            <td mat-cell *matCellDef="let element">
+                {{ element.positivePoint }}
+            </td>
+        </ng-container>
+        <ng-container matColumnDef="name">
+            <th mat-header-cell *matHeaderCellDef>
+                {{ 'VISIT_TABLE.NAME' | translate }}
+            </th>
+            <td mat-cell *matCellDef="let element">
+                {{ element.name }}
+            </td>
+        </ng-container>
+        <ng-container matColumnDef="type">
+            <th mat-header-cell *matHeaderCellDef>
+                {{ 'VISIT_TABLE.TYPE' | translate }}
+            </th>
+            <td mat-cell *matCellDef="let element">
+                {{ element.type }}
+            </td>
+        </ng-container>
 
-    <ng-container matColumnDef="image">
-        <th mat-header-cell *matHeaderCellDef>
-            {{ 'VISIT_TABLE.IMAGE' | translate }}
-        </th>
-        <td mat-cell *matCellDef="let element">
-            {{ element.image }}
-        </td>
-    </ng-container>
-</table>
+        <ng-container matColumnDef="image">
+            <th mat-header-cell *matHeaderCellDef>
+                {{ 'VISIT_TABLE.IMAGE' | translate }}
+            </th>
+            <td mat-cell *matCellDef="let element">
+                {{ element.image }}
+            </td>
+        </ng-container>
+        <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+        <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
+    </table>
+    <mat-paginator
+        [length]="100"
```

Comment:
`length` what is this? do we need it?

Reply:
no and i remove it

Reply:
![obraz](https://user-images.githubusercontent.com/79154804/233173493-3eb21e88-733a-4b55-8533-84190e30b137.png)
this is a definition